tasks/section1.yml

```diff
@@ -457,7 +457,7 @@
 # See not_automated.yml
 
 - name: "SCORED | 1.1.14 | AUDIT | Ensure nodev option set on /home partition"
-  command: /bin/true
+  shell: mount | grep /home | grep nodev
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*